app/javascript/packs/receiver.jsx

```diff
@@ -54,7 +56,15 @@ export default class Receiver extends React.PureComponent {
   }
 
   cancelUpdate() {
-    this.componentWillReceiveProps(this.props);
+    if (this.props.id === null) {
```

Comment:
If you're gonna ternary the other boi up, y'all wanna ternary this if / else up too?

Reply:
Mind if I don't? I like using ternaries in assignments but when both sides of the ternary are voids, to me it feels like it makes more sense as regular control flow

Reply:
😭 sure lol